api/config/cdr_versions_stable.json

```diff
@@ -11,7 +11,7 @@
     "publicDbName": "public20180831"
   },
   {
-    "name": "Test Registered CDR 20180920",
+    "name": "Synthetic Dataset v2",
```

Comment:
@danqrodney this one isn't "releaseNumber=2", should it be? I don't have context on this field.

Reply:
Release number is there as an ordering of releases; in future we expect to have both registered and controlled CDR versions with the same release number, released around the same time. For now it doesn't mean a whole lot.

Reply:
Got it. Is there a reason test is different from local/staging/stable here? (test sets releaseNumber=2)